en/docs/next/ai-workspace/mcp-proxies/configure-proxy.md

```
@@ -0,0 +1,117 @@
+
+## Get Started
+
+Once the proxy is deployed, you will see the MCP Proxy URL in the overview page.
+
+### Connect with an MCP Client
+
+Select a gateway from the **Gateways** dropdown to see the base URL for calling this proxy through that gateway.
+
+The URL follows the format:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)